user-guide/Reference/DataMiner_Tools/Ping_Monitoring/Working_with_Ping_Monitoring.md

```diff
@@ -0,0 +1,121 @@
+
+## The Ping Monitoring tool user interface
+
+To access the Ping Monitoring tool:
+
+1. Go to `http(s)://[DMA name]/root`.
+
+1. Select *PING Monitor* to start using the application.
+
+   ![PING Monitor](~/user-guide/images/Ping_Monitor.png)
+
+The Ping Monitoring tool UI consists of the following main components:
+
+- [Header bar](#the-ping-monitoring-header-bar) (1)
+
+- [Sidebar](#the-ping-monitoring-sidebar) (2)
+
+- [Overview pane](#the-ping-monitoring-overview-pane) (3)
+
+![Ping Monitoring UI](~/user-guide/images/PingMonitoring_UI.png)
+
+### The Ping Monitoring header bar
+
+You can find the following functionalities in the header bar of the application:
+
+- **Refresh**: Available only in the [*Destinations* tab](#the-destinations-tab). Allows you to update the destinations available in your DataMiner System.
+
+- **Alarm History**: Available only in the [*Alarms* tab](#the-alarms-tab). Allows you to access all alarms that occurred in the last 15 minutes, hour, 3 hours, or 24 hours.
+
+### The Ping Monitoring sidebar
+
+The sidebar on the left of the Ping Monitoring tool contains buttons that can be used to expand different panes:
+
+- ![Destinations](~/user-guide/images/Destinations_PM.png) : Clicking the *Destinations* button displays a list overview of all available destinations in your DataMiner System, along with all available information. See [The *Destinations* tab](#the-destinations-tab).
+
+- ![Grid View](~/user-guide/images/Grid_View.png) : Clicking the *Grid View* button displays a grid overview of all available destinations in your DataMiner System. See [The *Grid View* tab](#the-grid-view-tab).
+
+- ![Groups](~/user-guide/images/Groups_PM.png) : Clicking the *Groups* button displays an overview of all active groups in your DataMiner System. Each group corresponds to a Generic Ping element. See [The *Groups* tab](#the-groups-tab).
+
+- ![Alarms](~/user-guide/images/Alarms_PM.png) : Clicking the *Alarms* button displays an overview of all active alarms in the groups. See [The *Alarms* tab](#the-alarms-tab).
+
+### The Ping Monitoring overview pane
+
+#### The 'Destinations' tab
+
+The *Destinations* tab offers an overview of all internet destinations in your DataMiner System receiving ping commands on a predefined, customized basis.
+
+The following information is available for each destination:
+
+- **Destination address**: The URL or IP address of an internet destination, e.g. *www.skyline.be*.
+
+- **Description**: A brief description provided to the destination, e.g. *Skyline*.
+
+- **Group**: The group the destination belongs to, corresponding to a Generic Ping element, e.g. *Websites*.
+
+- **Customer**: The customer associated with the destination, e.g. *Skyline Communications*.
+
+- **Class**: The destination's provided class, e.g. *Bronze*.
+
+- **Region**: The destination's provided region, e.g. *US East*.
```

Comment:
Should "customer", "class", and "region" be removed here as well, @migbarq?

Reply:
Yes please, those properties were removed from the package because they caused issues when the customer did not have the customer properties created.